mne_bids/tests/test_write.py

```diff
@@ -503,3 +505,89 @@ def test_set():
 
     cmd = bids_validator_exe + [output_path]
     run_subprocess(cmd, shell=shell)
+
+
+@requires_nibabel()
+def test_write_anat():
+    """Test writing anatomical data."""
+    import nibabel as nib
+    # Get the MNE testing sample data
+    output_path = _TempDir()
+    data_path = testing.data_path()
+    raw_fname = op.join(data_path, 'MEG', 'sample',
+                        'sample_audvis_trunc_raw.fif')
+
+    event_id = {'Auditory/Left': 1, 'Auditory/Right': 2, 'Visual/Left': 3,
+                'Visual/Right': 4, 'Smiley': 5, 'Button': 32}
+    events_fname = op.join(data_path, 'MEG', 'sample',
+                           'sample_audvis_trunc_raw-eve.fif')
+
+    raw = mne.io.read_raw_fif(raw_fname)
+    write_raw_bids(raw, bids_basename, output_path, events_data=events_fname,
+                   event_id=event_id, overwrite=False)
```

Comment:
use `pytest.fixtures` to avoid code duplication here?

Reply:
I'd wait for this to be solved in #212